_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -0,0 +1,1028 @@
+				optionsLink: '#/settings?term=image%20optimize',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { 'photon-cdn': ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	'site-stats': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Site Stats' ),
+				details: __( 'Track your site visitors and learn about your most popular content.' ),
+				checked: getSetting( state, 'stats' ),
+				optionsLink: '#/settings?term=site%20stats',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { stats: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	'site-verification': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Site verification' ),
+				details: __( 'Verify your site with Google, Bing, Yandex, and Pinterest.' ),
+				checked: getSetting( state, 'verification-tools' ),
+				configureLink: '#/settings?term=verify',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { 'verification-tools': ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	subscriptions: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Subscriptions' ),
+				details: __( 'Send post notifications to your visitors.' ),
+				checked: getSetting( state, 'subscriptions' ),
+				optionsLink: '#/settings?term=subscriptions',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( updateSettings( { subscriptions: ! currentCheckedValue } ) );
+				},
+			};
+		},
+	},
+
+	testimonials: {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Testimonial: Custom content type' ),
+				details: __( 'Add testimonials to your website to attract new customers.' ),
+				checked: getSetting( state, 'jetpack_testimonial' ),
+				optionsLink: '#/settings?term=testimonials',
+			};
+		},
+		mapDispatchToProps: dispatch => {
+			return {
+				onToggleChange: currentCheckedValue => {
+					return dispatch( ( dispatchProp, getState ) => {
+						const jetpack_portfolio = getSetting( getState(), 'jetpack_portfolio' );
+						const jetpack_testimonial = ! currentCheckedValue;
+						const customContentTypes = jetpack_portfolio || jetpack_testimonial;
+
+						return dispatchProp(
+							updateSettings( {
+								jetpack_testimonial,
+								'custom-content-types': customContentTypes,
+							} )
+						);
+					} );
+				},
+			};
+		},
+	},
+
+	'tiled-galleries': {
+		mapStateToProps: state => {
+			return {
+				title: __( 'Tiled Galleries' ),
+				details: 'Add beautifully laid out galleries as a Gutenberg block',
+				checked: getSetting( state, 'tiled-gallery' ),
+				optionsLink: 'https://jetpack.com/support/jetpack-blocks/tiled-gallery-block/',
```

Comment:
I'd recommend using `getRedirectUrl` for links that leave the dashboard to go to Jetpack.com now.